services/cronback-scheduler-srv/src/spinner/controller.rs

```diff
@@ -186,6 +186,12 @@ impl SpinnerController {
         {
             let name_cache = self.trigger_name_cache.read().unwrap();
             if let Some(trigger_id) = name_cache.get(name) {
+                info!(
+                    trigger_id = ?trigger_id,
+                    project_id = ?project_id,
+                    name = name,
+                    "Found trigger with name cache"
+                );
```

Comment:
Isn't that a bit noisy for an "INFO" level? Not sure it also adds a lot of value